packages/client/utils/parseEmailAddressList.ts

```diff
@@ -21,33 +13,45 @@ const parseEmailAddressList = (rawStr = '') => {
     .replace(/,$/g, '')
 
   const emailCount = commaDelimStr.split(',').length
-  const lastChar = rawStr.charAt(rawStr.length - 1)
-  const isAddingNewEmail = lastChar !== ' ' && lastChar !== ','
+  const lastEmailIndex = commaDelimStr.lastIndexOf(' ' || ',')
```

Comment:
this doesn't do what i think you want it to. `' '` is always truthy so it'll never fall back to `','`

Reply:
Ah yes, thanks - just pushed an update